website/content/docs/getting-started/connect-to-target.mdx

```diff
@@ -7,15 +7,34 @@ description: |-
 
 ## Connect to Your First Target
 
-The default target is a `tcp` target with a default port of `22` (which
-corresponds to the default SSH port using TCP). The host sets for this target
-contain the default host, which has the address `127.0.0.1`. When `boundary
-connect` is executed against this target, the single available host will be
-selected and a local authenticated proxy to the host is established on the
-target's default port (`127.0.0.1:22`).
+The [Quick setup
+wizard](/docs/getting-started/deploy-and-login#login-to-hcp-boundary)
+is the fastest way to onboard a target to HCP Boundary.
 
+![Setup Wizard](/img/setup-wizard-target.png)
+
+A `tcp` target with a default port of `22` (the default SSH port using TCP) is
+recommended for an initial target. If using the Quick setup, a sample host set
+called **QA Databases** will contain the **CRM Database** host, with the Address
+provided during setup. When `boundary connect` is executed against the **CRM
+Database target**, the single available host will be selected and a local
+authenticated proxy to the host is established on the target's default port.
+
+![Setup Wizard](/img/quick-start-targets.png)
+
+Copy the **CRM Database target ID** (such as `ttcp_eTcZMueUYv`).
```

Comment:
Perhaps a good idea to apply step format here.
1. Copy the CRM Database target ID.
2. Open a terminal sessions.
3. Connect to the target

Reply:
Updated the format here, great suggestion